source/gui/__init__.py

```diff
@@ -75,6 +80,13 @@ def __init__(self):
 				self.Show()
 				self.Hide()
 
+	def _onDestroy(self, evt: wx.EVT_WINDOW_DESTROY):
+		self.Skip(skip=False)  # we should block the destruction events until the sysTrayIcon is destroyed

```

Comment:
"until" makes it sound like this logic will change at some point.

Hopefully I'm not alone in finding `skip` hard to memorize, I wish they had named it in the affirmative eg `handled`. 

The docs say:
>This method can be used inside an event handler to control whether further event handlers bound to this event will be called after the current one returns.
>
>Without Skip (or equivalently if Skip(false) is used), the event will not be processed any more. If Skip(true) is called, the event processing system continues searching for a further handler function for this event, even though it has been processed already in the current handler.
>
>In general, it is recommended to skip all non-command events to allow the default handling to take place. The command events are, however, normally not skipped as usually a single command such as a button click or menu item selection must only be processed by one handler.

This sounds like it will do the opposite of what the comment here says. 

**I've just noticed that this calls `self.Skip` rather than `evt.Skip` I assume this is a mistake?**

Reply:
I did find this hard to parse. I'm not sure what the default handler does for a `wx.EVT_WINDOW_DESTROY` event, but this is a command event so this behaviour is recommended and crashes otherwise. This event fires just before `self.Destroy` - which is what the comment was trying to allude to in a misleading way. I think the comment where the event handler is being bound should makes this clear anyway. 

And yes, the self/evt typo was a mistake.